pkg/cmd/migrate.go

```diff
@@ -50,18 +50,18 @@ func migrateRun(cmd *cobra.Command, args []string) error {
 		var err error
 		migrationDriver, err := crdbmigrations.NewCRDBDriver(dbURL)
 		if err != nil {
-			log.Fatal().Err(err).Msg("unable to create migration driver")
```

Comment:
move away from `log.Fatal` as it calls `os.Exit` and thus fails to unwind the call stack. `main.go` already does this if an `error` is returned by the cobra command.